assessment.md

```diff
@@ -13,81 +13,512 @@
+  * I = Rarely participate in the decision making process and share your opinion
+  * A = Sometimes participate in the decision making process and share your opinion
+  * G = Regularly participate in the decision making process and share your opinion
+  * E = Always participate in the decision making process and share your opinion.
 * Postmortems
+  * N = Never participate the analysis or discussion after the work is done
+  * I = Rarely participate the analysis or discussion after the work is done
+  * A = Sometimes participate the analysis or discussion after the work is done
+  * G = Regularly participate the analysis or discussion after the work is done
+  * E = Always participate the analysis or discussion after the work is done
 * Conflicts
+  * N = Never handles differences with teammates in mature manner
+  * I = Rarely handles differences with teammates in mature manner
+  * A = Sometimes handles differences with teammates in mature manner
+  * G = Regularly handles differences with teammates in mature manner
+  * E = Always handles differences with teammates in mature manner
 * Infractions
+  * N = Always breaks the team rules
+  * I = Regularly breaks the team rules
+  * A = Sometimes breaks the team rules
+  * G = Rarely breaks the team rules
+  * E = Never breaks the team rules
 
 ### Interactions
 
 * Interaction with the customer
+  * N = Does not respect the customer and rejects all their ideas
+  * I = Respect customer but rejects most of their ideas
+  * A = Respect customer but fails to build a system around their needs
+  * G = Respect customer and builds a system around their needs but fails
+     to clearly present your system to your customer
+  * E = Respect customer and builds a system around their needs while being
+     able to clearly present your system to them
 * Interaction with the team leaders
+  * N = Impolite and impatient
+  * I = Polite but impatient
+  * A = Polite and patient but does not have clear requests
+  * G = Polite and patient but does not respond to team leaders
+  * E = Polite and patient
 * Interaction with the course instructor
+  * N = Communicated with the instructor badly, did not follow
+     the project and course requirements
+  * I = Communicated with the instructor poorly, followed part
+     of the project and course requirements
+  * A = Communicated with the instructor well, followed most
+     of the project and course requirements
+  * G = Communicated with the instructor well, followed all
+     project and course requirements
+  * E = Communicated with the instructor excellently, followed
+     all project and course requirements
 * Interaction with the teaching assistants
+  * N = Communicated with the TAs badly, hate speech, ignore
+     TAs when they are trying to help
+  * I = Communicated with the TAs poorly, did not use them well as your
+     resources, did not talk to them
+  * A = Communicated with the TAs well, but relied on them too much, asked
```

Comment:
"most of the work"? That should be much lower on the grading scale.

Reply:
I just changed "most of the work" to "too many works". Does that work?

Reply:
No, @Lancasterwu, it should be "too much work".